Data/Streaming/Network.hs

```diff
@@ -138,10 +139,21 @@ getSocketFamilyGen sockettype host' port' af = do
 getSocketGen :: SocketType -> String -> Int -> IO (Socket, AddrInfo)
 getSocketGen sockettype host port = getSocketFamilyGen sockettype host port NS.AF_UNSPEC
 
+defaultSocketOptions :: SocketType -> [(NS.SocketOption, Int)]
+defaultSocketOptions sockettype =
+    case sockettype of
+        NS.Datagram -> [(NS.ReuseAddr,1)]
+        _           -> [(NS.NoDelay,1), (NS.ReuseAddr,1)]
+
 -- | Attempt to bind a listening @Socket@ on the given host/port using given
 -- @SocketType@. If no host is given, will use the first address available.
 bindPortGen :: SocketType -> Int -> HostPreference -> IO Socket
-bindPortGen sockettype p s = do
+bindPortGen sockettype = bindPortGenEx (defaultSocketOptions sockettype) sockettype
+
+-- | Attempt to bind a listening @Socket@ on the given host/port using given
+-- socket options and @SocketType@. If no host is given, will use the first address available.
```

Comment:
Can you add a `@since` comment, add a comment to the ChangeLog, and bump the version in the .cabal file (this would be a minor version bump)?